pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/controversial/xml/UnnecessaryParentheses.xml

```diff
@@ -1,88 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<test-data>
-    <test-code>
```

Comment:
Ok! assignment and return statements are now treated equally, so I guess that's alright!